configmap/configmap.go

```diff
@@ -69,13 +50,13 @@ func VerifyCM(configMap *corev1.ConfigMap, verifySteps []config.VerifyStep) (map
 				if err != nil {
 					return verifiedFiles, "", err
 				}
-				err = writeOSFile(tempFile, []byte(fileContents))
+				err = filesystem.WriteOSFile(tempFile, []byte(fileContents))
 				if err != nil {
 					return verifiedFiles, "", err
 				}
 
 				// In the end just remove the temporary file, regardless of the verification result
-				defer deleteFile(tempFile.Name())
+				defer filesystem.DeleteFile(tempFile.Name())
```

Comment:
This defer should be right beneath the creation of the temporary file. This defer will miss a return in line 55.

Reply:
fixed